pysqream_sqlalchemy/dialect.py

```diff
@@ -164,13 +155,17 @@ def get_columns(self, connection, table_name, schema=None, **kwargs):
             col_name = col_meta[1]
             try:
                 type_key = col_meta[2].split()[0].split('(')[0]
+                if "[]" in type_key:
+                    logging.warning(f"Arrays have not supported yet. Column {col_name} with type {type_key} has been skipped")
+                    continue
```

Comment:
@eugenek-sqream I think that we need to throw an error. what do you think?

Reply:
good idea, raise exception added